openpathsampling/high_level/move_scheme.py

```diff
@@ -721,6 +722,10 @@ def move_summary(self, steps, movers=None, output=sys.stdout, depth=0):
         depth : integer or None
             depth of submovers to show: if integer, shows that many
             submovers for each move; if None, shows all submovers
+        scheme_copies : lits of :class:`MoveScheme`
```

Comment:
I think that is an excellent choice. So we can keep the two things separated as we do with visualisation (which is also some kind of analysis). So let's keep this for now.